ext/afform/core/Civi/Api4/Action/Afform/Submit.php

```diff
@@ -352,7 +352,7 @@ public static function processGenericEntity(AfformSubmitEvent $event) {
       catch (\CRM_Core_Exception $e) {
         // What to do here? Sometimes we should silently ignore errors, e.g. an optional entity
         // intentionally left blank. Other times it's a real error the user should know about.
-        \Civi::log('afform')->debug("Silently ignoring exception in Afform processGenericEntity call: " . $e->getMessage());
+        \Civi::log('afform')->debug('Silently ignoring exception in Afform processGenericEntity call for "' . $event->getEntityType() . '". Message: ' . $e->getMessage());
```

Comment:
I think `getEntityName()` would be even more helpful.

Reply:
Yes I agree!